### PR TITLE
Allow table stack breakpoint to be set with a variable

### PR DESCRIPTION
**Pull request by mwitek**
Adding a var to set the breakpoint for stacked tables.

### CODE REVIEW

File: scss/settings/_settings.scss [hunk 1]
```diff
@@ -556,6 +556,7 @@ $table-foot-background: smart-scale($table-background, $table-color-scale);
 $table-foot-row-hover: darken($table-foot-background, $table-hover-scale);
 $table-head-font-color: $body-font-color;
 $table-foot-font-color: $body-font-color;
+$table-stack-breakpoint: medium;
```
Comment (IamManchanda) [inline scss/settings/_settings.scss]: Setting files is generated automatically .....  

You need to add this variable here after this => https://github.com/mwitek/foundation-sites/blob/dbea7aa151d175f2f3db6bf7379d7bddec1d60c9/scss/components/_table.scss#L73

Please remove changes from settings!

**Review by IamManchanda — CHANGES_REQUESTED** [review]
Poke @kball Settings again :D 
#Dist

**Comment by mwitek** [comment]
@IamManchanda Ready for another look.

**Comment by kball** [comment]
Looks good to me!

**Pull request merged**